CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### 🛡 Security
 
+- [CVE-2022-1537][CVE-2022-0436]bump grunt from 1.4.1 to 1.5.3 ([#3723](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3723))
```

Comment:
I thought our convention was a separate line for each CVE, just so it's easy to scan/search?